src/components/tiles/workshop-tile.tsx

```diff
@@ -6,7 +6,8 @@ export function WorkshopTile() {
   return (
     <Tile className='sm:col-span-6 md:col-span-3'>
       <Tile.Body>
-        <h2 className='text-3xl sm:text-[34px]'>A 21. század műanyaga az AI</h2>
+        <h2 className='text-3xl sm:text-[34px] mb-1'>A 21. század műanyaga az AI</h2>
+        <p className='mb-2 text-[22px]'>{`IB110 | 14:30-16:30`}</p>
```

Comment:
Sorry, maybe I wasn't clear. This info should only be shown on the workshop details page, not on the tile

Reply:
And the same goes for the presentations?

Reply:
no, I think it can stay for the presentations, their tile is way bigger